internal/block/block.go

```diff
@@ -72,12 +67,12 @@ func (b *neoBlock) SetTransactions(txx []dbft.Transaction[crypto.Uint256]) {
 }
 
 // NewBlock returns new block.
-func NewBlock(timestamp uint64, index uint32, prevHash crypto.Uint256, version uint32, nonce uint64, txHashes []crypto.Uint256) dbft.Block[crypto.Uint256, crypto.Uint160] {
+func NewBlock(timestamp uint64, index uint32, prevHash crypto.Uint256, nonce uint64, txHashes []crypto.Uint256) dbft.Block[crypto.Uint256, crypto.Uint160] {
 	block := new(neoBlock)
 	block.base.Timestamp = uint32(timestamp / 1000000000)
 	block.base.Index = index
 	block.base.PrevHash = prevHash
-	block.base.Version = version
+	block.base.Version = 0 // Not a consensus subject.
```

Comment:
It's a consensus subject, but it's implementation-specific. We will have a new block version after https://github.com/neo-project/neo/issues/1526, but dBFT logic doesn't care much, it's all about particular proposal verification.

Reply:
Comment is fixed.

> We will have a new block version after https://github.com/neo-project/neo/issues/1526

Sure, I'm tracking this issue.